contrib/epee/src/CMakeLists.txt

```diff
@@ -26,11 +26,19 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+set(EPEE_INCLUDE_DIR_BASE "${CMAKE_CURRENT_SOURCE_DIR}/../include")
+
+file(GLOB EPEE_HEADERS_PUBLIC 
+	"${EPEE_INCLUDE_DIR_BASE}/*.h*"
+	"${EPEE_INCLUDE_DIR_BASE}/**/*.h*"
```

Comment:
Is ** a special case for "any number of subdirectories" ?

Reply:
Yes. [Here's](https://stackoverflow.com/a/62985520) some discussion.

Shall I simply explain this in a comment?

Reply:
Up to you, I think I've seen this kind of syntax elsewhere before, it seems fairly self explanatory there even if you don't know what it does, you can guess.